### PR TITLE
Thesilence docs

### PR DESCRIPTION
**Pull request by therealsilence**
- Add Synapse data model background on light edges
- Update Storm data mod and pivot references for light edges
- Update Storm command reference for additional cmds
- Glossary updates
- Minor updates / typo fixes

### CODE REVIEW

File: docs/synapse/glossary.rst [hunk 1]
```diff
@@ -229,6 +229,16 @@ Edge, Directed
 In a :ref:`gloss-directed-graph`, a directed edge is used to connect exactly two nodes (vertexes) in a one-way
 (directional) relationship. Compare with :ref:`gloss-hyperedge`.
 
+.. _gloss-edge-light:
+
+Edge, Lightweight (Light)
+-------------------------
+
+In Synapse, a lightweight (light) edge is a model element that links two arbitrary forms via a user-defined
```
Comment (vEpiphyte) [inline docs/synapse/glossary.rst:237]: It's technically not a model element, since its not a *node* per say.

File: docs/synapse/glossary.rst [hunk 2]
```diff
@@ -229,6 +229,16 @@ Edge, Directed
...
+verb that describes the linking relationship. Light edges are not forms and so do not support secondary 
+properties or tags. They are meant to simplify performance, represenation of data, and Synapse hypergraph
```
Comment (vEpiphyte) [inline docs/synapse/glossary.rst:239]: spelling: representation

File: docs/synapse/glossary.rst [hunk 3]
```diff
@@ -280,6 +290,21 @@ In the Synpase :ref:`gloss-data-model`, a category of form whose primary propert
...
+.. _gloss-form-digraph:
+
+See :ref:`gloss-form-edge`.
+
+.. _gloss-form-edge:
+
+Form, Edge
+----------
+
+In the Synappse :ref:`gloss-data-model`, a specialized **composite form** (:ref:`gloss-form-comp`) whose primary
```
Comment (Cisphyx) [inline docs/synapse/glossary.rst:302]: "Synappse"

File: docs/synapse/userguides/bkd_why_synapse.ipynb [hunk 4]
```diff
@@ -76,10 +76,10 @@
...
     "\n",
-    "Automated Tasking Subsystem\n",
-    "---------------------------\n",
+    "Automation\n",
+    "----------\n",
     "\n",
-    "Synapse supports automation through the use of **triggers** that can perform arbitrary actions upon the occurrence of a specific event (such as adding a node or applying a tag) as well as a cron scheduling system. Both features leverage the Storm query language, and can carry out any action that can be expressed as a Storm query.\n",
+    "Synapse supports automation through a number of features, including a cron scheduling system, **triggers** that can perform arbitrary actions upon the occurrence of a specific event (such as adding a node or applying a tag), and stored macros. All three features leverage the Storm query language, and can carry out any action that can be expressed as a Storm query.\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/bkd_why_synapse.ipynb:82]: we also support runtime storm daemons

File: docs/synapse/userguides/storm_ref_cmd.ipynb [hunk 5]
```diff
@@ -1232,6 +1541,135 @@
...
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _storm-model-edge-set:\n",
+    "\n",
+    "model.edge.set\n",
+    "++++++++++++++\n",
+    "\n",
+    "The ``model.edge.set`` command allows you to set the value of an extended property on a light edge (such as a ``:doc`` property to specify a definition for the light edge). The extended property must already exist / be defined within Synapse.\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_cmd.ipynb:1601]: we'll need to reword this per discussion

File: docs/synapse/userguides/storm_ref_data_mod.ipynb [hunk 6]
```diff
@@ -502,6 +504,204 @@
...
+    "podes = await core.eval(q1, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "**Usage Notes:**\n",
+    "\n",
+    "- No light edge verbs exist in a Cortex by default; they must be created.\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.ipynb:611]: probably want a note about required permissions to make a light edge

File: docs/synapse/userguides/storm_ref_data_mod.ipynb [hunk 7]
```diff
@@ -502,6 +504,204 @@
...
+    "  - The Storm :ref:`storm-model` commands can be used to list and work with any light edge verbs in a Cortex.\n",
+    "\n",
+    "- A light edge's verb typically has a logical direction (a report \"references\" a set of indicators that it contains, but the indicators do not \"reference\" the report). However, it is up to the user to create the light edges in the correct direction and using forms that are sensical for the light edge verb. That is, there is nothing in the Storm syntax iitself to prevent users linking any arbitrary nodes in arbitrary directions using arbitrary light edge verbs.\n",
+    "- The plus sign ( ``+`` ) used with the light edge expression within the edit brackets is used to create the light edge(s).\n",
+    "- Light edges can be created in either \"direction\" (e.g., with the directional arrow pointing either right ( ``+(<verb>)>`` ) or left ( ``<(<verb>)+`` ) - whichever syntax is easier.\n",
+    "- The nodes referenced by the Storm expression (i.e., within the curly braces) are **NOT** created by the syntax above. They must already exist in order for the light edges to be created."
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.ipynb:620]: the storm expression is arbitrary and can be used to make nodes or do anything else. the nodes emitted are the ones linked.

File: docs/synapse/userguides/storm_ref_data_mod.ipynb (hunk 6 above)
Comment (therealsilence) [reply docs/synapse/userguides/storm_ref_data_mod.ipynb:611]: This is true for everything though - you need perms to delete a node, set a tag, etc. Would rather leave perms discussion to the getting started / devops docs.

File: docs/synapse/userguides/storm_ref_cmd.ipynb (hunk 5 above)
Comment (vEpiphyte) [reply docs/synapse/userguides/storm_ref_cmd.ipynb:1601]: i will update this

File: docs/synapse/userguides/storm_ref_cmd.ipynb [hunk 8]
```diff
@@ -1232,6 +1541,135 @@
...
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _storm-model-edge-get:\n",
+    "\n",
+    "model.edge.get\n",
+    "++++++++++++++\n",
+    "\n",
+    "The ``model.edge.get`` command allows you to retrieve the value of an extended property on a light edge (such as a ``:doc`` property to specify a definition for the light edge).\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_cmd.ipynb:1628]: I will update this

File: docs/synapse/userguides/storm_ref_cmd.ipynb [hunk 9]
```diff
@@ -1232,6 +1541,135 @@
...
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _storm-model-edge-del:\n",
+    "\n",
+    "model.edge.del\n",
+    "++++++++++++++\n",
+    "\n",
+    "The ``model.edge.del`` command allows you to delete an extended property from a light edge (such as a ``:doc`` property to specify a definition for the light edge). Deleting an extended property from a specific light edge does not delete the property from Synapse (e.g., the property can be re-added to the light edge or to other light edges).\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_cmd.ipynb:1655]: i will update this

**Pull request merged**